docs/release-planning.md

```
@@ -0,0 +1,189 @@
+ℹ️  Dry run - no approval process initiated
+```
+
+### Major Release with Multiple Components
+```bash
+tgk release plan --type major --components core,ui,workers,database --urgency high
+```
+
+This will:
+- Generate comprehensive release plan with 10+ steps
+- Assess as high risk
+- Require 4 approvals (2 Tech Lead, 1 Security, 1 Product)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)